crm_claim_code/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from . import models
+from openerp import SUPERUSER_ID
+
+
+def assign_old_sequences(cr, registry):
+    cr.execute("""
```

Comment:
Neither do I :worried: 

I think it was because of the module renaming, maybe I should take the code that was deleted.

https://github.com/oihane/crm/commit/ac27837143d94fbcba2afed724f5db24a33bf284#diff-395acdde16ff09e591a67b8ee4661d9cL23